docker-compose.yaml

```diff
@@ -28,6 +28,22 @@ services:
     init: true
     ipc: host
 
+  nginx:
+    image: nginx:1.18.0
+    env_file:
+      - env/.nginx.env
+    ports:
+      - "80:80"
+      - "443:443"
```

Comment:
Not sure these ports should be exposed in this `docker-compose.yaml` - for example for staging we need port 8080, no?

So in this base `docker-compose.yaml` we maybe don't want to expose any ports, and then do it in
- .override.yaml for local development
- .staging.yaml for the staging env
- .production.yaml for the production env

Reply:
Added in ce6a6e303ba84582282ee3ac9d3799330c945031